app/controllers/admin/userManagement.py

```diff
@@ -40,9 +40,31 @@ def manageUsers():
         else:
             removeCeltsStudentStaff(user)
             flash(username+ " is no longer a Celts Student Staff", 'success')
-
     return ("success")
 
+@admin_bp.route('/updateManagers', methods=['POST'])
+def updateProgramManagers():
+    eventData = request.form
+    if  int(eventData['status']) == 0:
+        try:
+            addProgramManager(eventData['userID'],int(eventData['programID']))
+        except:
+            flash('Error while trying to add a manager.')
+            return "Error while adding manager."
+
+    elif int(eventData['status']) == 1:
+        try:
+            removeProgramManager(eventData['userID'],int(eventData['programID']))
+        except:
+            flash('Error while trying to remove a manager.')
+            return "Error while removing manager."
+
+    else:
+        flash('Error while removing a manager.')
+        return "Error while removing manager."
```

Comment:
All of these errors should print the actual error to the console so we can debug the problem. Right now you are catching the exceptions and not telling anyone except the user.

Also, errors should return an appropriate http status code. Before you were returning 500 for everything. Now you are returning 200 for everything. Neither is correct.